pkg/crd/schema.go

```diff
@@ -427,6 +427,10 @@ func builtinToType(basic *types.Basic, allowDangerousTypes bool) (typ string, fo
 		format = "int32"
 	case types.Int64, types.Uint64:
 		format = "int64"
+	case types.Float32:
+		format = "float"
+	case types.Float64, types.UntypedFloat:
+		format = "double"
```

Comment:
Hi @tamalsaha, 

That is nice. But did you check the comment on top of this method? See:

```
// builtinToType converts builtin basic types to their equivalent JSON schema form.
// It *only* handles types allowed by the kubernetes API standards. Floats are not
// allowed unless allowDangerousTypes is true
```
 
In this way, I understand that we cannot add the floats here.